openjdk/src/test/java/org/conscrypt/AbstractSessionContextTest.java

```diff
@@ -16,16 +16,25 @@
 
 package org.conscrypt;
 
+import com.sun.org.apache.xerces.internal.impl.dv.util.Base64;
```

Comment:
Thanks for noticing! This redundant import was left over from my testing. Fixed now.